tools/clang/unittests/HLSLExec/LongVectors.cpp

```diff
@@ -1146,7 +1223,7 @@ STRICT_OP_1(OpType::LoadAndStore_RD_SB_SRV, (A));
 
 template <OpType OP, typename T> struct ExpectedBuilder {
 
-  static auto buildExpected(Op<OP, T, 1> Op, const InputSets<T> &Inputs) {
+  static auto buildExpected(Op<OP, T, 1> &Op, const InputSets<T> &Inputs) {
```

Comment:
Wondering why some of the `buildExpected`'s were updated to take the non-const reference, even though they don't actually want to modify the op?

Reply:
Felt weird to have differing signatures between different versions of buildExpected. But now that you mention it's cleaner to only update where its needed.

Reply:
I figured that's what it was - but you hadn't updated all of the different versions :)